siliconcompiler/tools/vpr/show.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+from siliconcompiler.tools.vpr import vpr
+
+
+def setup(chip, clobber=True):
+    '''
+    Show placed and/or routed designs in VPR GUI
+    '''
+
+    tool = 'vpr'
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    task = chip._get_task(step, index)
+
+    vpr.setup_tool(chip, clobber=clobber)
+
+    chip.set('tool', tool, 'task', task, 'threads', os.cpu_count(),
+             step=step, index=index, clobber=False)
+
+
+def runtime_options(chip):
+    '''Command line options to vpr for show
+    '''
+
+    options = generic_show_options(chip)
+
+    options.append('--disp on')
+
+    return options
+
+
+def generic_show_options(chip):
+    ''' Helper function to setup options for show and screenshot
+    '''
+
+    step = chip.get('arg', 'step')
+    index = chip.get('arg', 'index')
+    tool, task = chip._get_tool_task(step, index)
+
+    options = vpr.runtime_options(chip)
+
+    if chip.valid('tool', tool, 'task', task, 'var', 'show_filepath'):
+        show_job = chip.get('tool', tool, 'task', task, 'var', 'show_job',
+                            step=step, index=index)
+        show_step = chip.get('tool', tool, 'task', task, 'var', 'show_step',
+                             step=step, index=index)
```

Comment:
if this is false none of these variables will be set

Reply:
Done